third_party/xla_client/computation_client.cc

```diff
@@ -328,6 +305,22 @@ std::vector<std::string> ComputationClient::GetCompilationDevices(
   return compilation_devices;
 }
 
+void ComputationClient::RunLocalService() {
+  try {
+    XrtLocalService* service =
+        new XrtLocalService("localservice|localhost:51011", "localservice", 0);
```

Comment:
yea, I can add that option.

Reply:
on a second thought, we also need to update our `xla_dist` to take port number for `XRT_TPU_CONFIG`(currently fixed to 51011). If user changed the port number, they also need to force a server restart(by passing the flag to xla_dist), otherwise new server won't be open and client will try to connect to the tf server in a incorrect port.